test/resetPassword.test.js

```diff
@@ -0,0 +1,146 @@
+const chai = require('chai');
+const chaiHTTP = require('chai-http');
+const { app } = require('../app');
+const { expect } = require('chai');
+import { v4 as uuid } from 'uuid';
+import { resetTokens, users } from '../models';
+const db = require('../models');
+
+chai.should();
+chai.use(chaiHTTP);
+
+describe('Reset password : All', () => {
+	before((done) => {
+		db.users.destroy({
+			where: {},
+			truncate: true,
+		});
+		done();
+	});
+	it('should create a User', async () => {
+		const user = {
+			fullname: 'cyifuzo jean chrysostome',
+			username: 'chance',
+			password: 'test123',
+			email: 'delcy@gmail.com',
+		};
+		const response = await chai.request(app).post(`/api/v1/users`).send(user);
+		expect(response).to.have.status(201);
```

Comment:
You can add a few more assertions, to see if you are getting the right response.